presto-main/src/test/java/com/facebook/presto/spiller/TestFileSingleStreamSpiller.java

```diff
@@ -64,11 +68,30 @@ public void tearDown()
     public void testSpill()
             throws Exception
     {
-        PagesSerdeFactory serdeFactory = new PagesSerdeFactory(new BlockEncodingManager(new TypeRegistry()), false);
-        PagesSerde serde = serdeFactory.createPagesSerde();
-        SpillerStats spillerStats = new SpillerStats();
+        testSpillWithSettings(false);
+    }
+
+    @Test
+    public void testSpillCompression()
+            throws Exception
+    {
+        testSpillWithSettings(true);
+    }
+
+    private void testSpillWithSettings(boolean compression)
```

Comment:
for consistency, consider renaming to assertSpill

Reply:
Renamed.